.github/workflows/test-CI.yml

```diff
@@ -0,0 +1,104 @@
+name: Test Library CI
+
+on:
+  # Trigger the workflow on push to the specific branch
+  push:
+    branches:
+      - dev
+      - main
+      - Nadav/CAD-22801/create_CI-library_workflow
+
+  # Trigger the workflow on pull requests targeting the specific branch
+  pull_request:
+    branches:
+      - dev
+      - main
+      - Nadav/CAD-22801/create_CI-library_workflow
```

Comment:
Remove your branch from here

Reply:
It is there only for testing purposes - will be eliminated later on